src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -245,19 +195,17 @@ protected int doWork() {
         String title = INPUT.getName().replace(".duplicates_marked", "").replace(".aligned.bam", "");
 
         // Qualify the title with the library name iff it's for a single sample
-        final List<SAMReadGroupRecord> readGroups = sam.getFileHeader().getReadGroups();
+        final List<SAMReadGroupRecord> readGroups = save_header.getReadGroups();
         if (readGroups.size() == 1) {
```

Comment:
given that this is your only usage of the header, I'd pull out the information you need (during the initialize) and store it (it's a String) rather than holding on to an old header.